src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html

```diff
@@ -1,45 +1,10 @@
-<ng-template #usageTooltipTpl>
-  <table *ngIf="!showMultisiteTooltip">
-    <tr>
-      <td class="text-left me-1">Used:</td>
-      <td class="text-right"><strong> {{ isBinary ? (used | dimlessBinary) : (used | dimless) }}</strong></td>
-    </tr>
-    <tr *ngIf="calculatePerc && showFreeToolTip">
-      <td class="text-left me-1">Free:</td>
-      <td class="'text-right"><strong>{{ isBinary ? (total - used | dimlessBinary) : (total - used | dimless) }}</strong></td>
-    </tr>
-    <tr *ngIf="customLegend">
-      <td class="text-left me-1">{{ customLegend }}:</td>
-      <td class="text-right"><strong>{{ isBinary ? (customLegendValue | dimlessBinary) : (customLegend[1] | dimless) }}</strong></td>
-    </tr>
-  </table>
-  <table *ngIf="showMultisiteTooltip">
-    <tr>
-      <td class="text-left">Total Shards:&nbsp;</td>
-      <td class="text-right"><strong> {{ total }}</strong></td>
-    </tr>
-    <tr *ngIf="calculatePerc">
-      <td class="text-left">Transferred Shards:&nbsp;</td>
-      <td class="'text-right"><strong>{{ used }}</strong></td>
-    </tr>
-  </table>
-</ng-template>
-
-<div class="progress"
-     data-placement="left"
-     [ngbTooltip]="usageTooltipTpl">
-  <div class="progress-bar bg-info"
-       [ngClass]="{'bg-warning': warningThreshold && (warningThreshold >= 0) && (usedPercentage/100 >= warningThreshold), 'bg-danger': errorThreshold && (errorThreshold >= 0) && (usedPercentage/100 >= errorThreshold)}"
-       role="progressbar"
-       [attr.aria-label]="{ title }"
-       i18n-aria-label="The title of this usage bar is { title }"
-       [style.width]="usedPercentage + '%'">
-    <span [style.color]="usedPercentage < 60 ? 'black' : 'white'">{{ usedPercentage | number: '1.0-' + decimals }}%</span>
-  </div>
-  <div class="progress-bar bg-freespace"
-       role="progressbar"
-       [attr.aria-label]="{ title }"
-       i18n-aria-label="The title of this usage bar is { title }"
-       [style.width]="freePercentage + '%'">
-  </div>
+<div>
+    {{ isBinary ? (used | dimlessBinary) : (used | dimless) }} / {{ isBinary ? (total | dimlessBinary) : (total | dimless) }}
```

Comment:
This does not follow carbon's examples:
![image](https://github.com/user-attachments/assets/68625fb4-4e9e-4bfd-94ea-ceb89070928a)

I think it should be:
```suggestion
<div>
    {{ isBinary ? (used | dimlessBinary) : (used | dimless) }} used ({{ isBinary ? (total | dimlessBinary) : (total | dimless) }} available)
```

Reply:
thanks, while making changes I had it removed somehow